pyre/check.py

```
@@ -1471,6 +1485,63 @@ def synth_skip_backends(path):
+        for _ in range(20):
+            line = source.readline()
+            if not line:
+                break
+            if not line.startswith(prefix):
+                continue
+            bands = {}
+            entries = line[len(prefix):].strip().split(",")
+            for entry in entries:
+                parts = entry.split("=")
+                if len(parts) != 2 or not all(part.strip() for part in parts):
+                    raise ValueError(f"invalid jit-stats band in {path}: {line.strip()}")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)